### DISCUSSION TITLE
Julia integration

### DISCUSSION THREAD

**Discussion by dpinol** [prometheus/prometheus#9378] [Ideas]
Hi,
are there plans to provide a client for Julia?
Has anybody used it from Julia maybe through API or through another language (c or python)?

thanks

**Comment by LeviHarrison** [comment]
I assume that such a client library wouldn't be official, but I'm surprised that there isn't any complete 3rd party library available yet.

**Reply by LeviHarrison** [reply]
All I can find is this: https://github.com/tmlbl/Prometheus.jl, but it seems to still be a WIP.
